customer.py

```diff
@@ -0,0 +1,73 @@
+import requests
+import datetime
+
+class Customer:
+    def __init__(self, url="http://localhost:5000", selected_customer=None):
+        self.url = url
+        self.selected_customer = selected_customer
+    
+    def create_customer(self, customer_name, postal_code, phone_number):
+        request_body = {
+            "name": customer_name,
+            "postal_code": postal_code,
+            "phone": phone_number
+        }
+        response = requests.post(self.url+"/customers",json=request_body)
+        return response.json()
+
+
+    def update_customer(self, customer_name=None, postal_code=None, phone_number=None):
+        if not name:
+            name = self.selected_customer["name"]
+        if not postal_code:
+            postal_code = self.selected_customer["postal_code"]
+        if not phone_number:
+            phone_number = self.selected_customer["phone"]
+
+        query_params = {
+        "name": customer_name,
+        "postal code": postal_code,
+        "phone": phone_number
+        }
+
+        response = requests.put(
+            self.url+f"/customers/{self.selected_customer['id']}", 
+            json=query_params)
+        return response.json()
+
+
+    def delete_customer(self):
+        response = requests.delete(self.url+f"/customers/{self.selected_customer['id']}")
```

Comment:
This function assumes that a video has been selected. When writing a class like this I recommend adding error checking to verify that selected_video is not None and has a field 'id' before accessing 'id', or wrapping this section in a try/except block to catch a potential error.